ui/admin/app/components/form/credential-store/vault/index.hbs

```diff
@@ -46,77 +46,47 @@
+            @maxWidth='20rem'
+            {{on 'input' (fn @changeType credentialStoreType)}}
+            as |R|
           >
-            <radioGroup.card
-              @label={{t 'resources.credential-store.types.static'}}
-              @helperText={{t 'resources.credential-store.help.static'}}
-              @value='static'
-              @layout='wide'
-            />
-            <radioGroup.card
-              @label={{t 'resources.credential-store.types.vault'}}
-              @helperText={{t 'resources.credential-store.help.vault'}}
-              @value='vault'
-              @layout='wide'
-            />
-          </form.radioGroup>
-        {{else}}
-          <form.radioGroup
-            @name='types'
-            @selectedValue={{@model.type}}
-            as |radioGroup|
-          >
-            <radioGroup.card
-              @label={{t
-                (concat 'resources.credential-store.types.' @model.type)
-              }}
-              @helperText={{t
-                (concat 'resources.credential-store.help.' @model.type)
-              }}
-              @value={{@model.type}}
-              @layout='wide'
-              readonly={{true}}
-              @disabled={{true}}
-            />
-          </form.radioGroup>
-        {{/if}}
-      </:body>
-    </form.fieldset>
-  {{else}}
-    <form.fieldset>
-      <:title>{{t 'form.type.label'}}</:title>
-      <:body>
-        <form.radioGroup
-          @name='types'
-          @selectedValue={{@model.type}}
-          as |radioGroup|
-        >
-          <radioGroup.card
-            @label={{t
-              (concat 'resources.credential-store.types.' @model.type)
-            }}
-            @helperText={{t
-              (concat 'resources.credential-store.help.' @model.type)
-            }}
-            @value={{@model.type}}
-            @layout='wide'
-            readonly={{true}}
-            @disabled={{true}}
-          />
-        </form.radioGroup>
-      </:body>
-    </form.fieldset>
+            <R.Label>{{t
+                (concat 'resources.credential-store.types.' credentialStoreType)
+              }}</R.Label>
+            <R.Description>{{t
+                (concat 'resources.credential-store.help.' credentialStoreType)
+              }}</R.Description>
+            {{#each-in type as |icon name|}}
+              {{#if name}}
+                <R.Icon @name={{name}} />
+              {{/if}}
+            {{/each-in}}
+          </G.RadioCard>
+        {{/each-in}}
+      </Hds::Form::RadioCard::Group>
+    {{else}}
+      <InfoField
+        @value={{t (concat 'resources.credential-store.help.' @model.type)}}
+        @icon={{@icon}}
+        as |F|
+      >
+        <F.Label>{{t 'form.type.label'}}</F.Label>
+        <F.HelperText>{{t
+            (concat 'resources.credential-store.types.' @model.type)
+          }}
+        </F.HelperText>
+      </InfoField>
+    {{/if}}
   {{/if}}
-
+  <br />
```

Comment:
This is adding extra space after the radio card group?  Can it be applied via the HDS override styles?  We add margin for some fields, but radio cards/groups may not be covered yet.